include/comingsoon.css

```diff
@@ -0,0 +1,10 @@
+.bgimg {
+background-image: url('http://www.dreamalittlebigger.com/wp-content/uploads/2013/08/gold-hi-res-glitter-freebie-dream-a-little-bigger.jpg?x32341');
```

Comment:
I do want to make a site for commercial purposes. I'll take you up on that discussion.